schemas/common/autogeneratedResources.json

```diff
@@ -2857,6 +2857,12 @@
         {
           "$ref": "https://schema.management.azure.com/schemas/2016-11-01/Microsoft.ImportExport.json#/resourceDefinitions/jobs"
         },
+        {
+          "$ref": "https://schema.management.azure.com/schemas/2018-05-01-preview/Microsoft.Insights.json#/resourceDefinitions/components"
```

Comment:
You shouldn't need to make any changes to this file - it's only for teams using the automatic generation pipeline. Please can you revert these changes?